c2corg_ui/templates/helpers/view.html

```diff
@@ -226,9 +226,19 @@ <h3 class="heading show-phone outings" ng-click="mainCtrl.animateHeaderIcon($eve
 
 
 <%def name="get_document_locale_text(locale, text)">\
+
+
   ## WEATHER AND CONDITIONS
   % if text == 'weather':
-    % if locale['weather'] or locale['conditions'] or locale['conditions_levels']:
+  <%
+      cond = json.loads(locale['conditions_levels'])
+      conditions_levels_exist = (len(cond) > 0) and \
+          ( ('level_snow_height_total' in cond[0] and cond[0].get('level_snow_height_total') != '' and cond[0].get('level_snow_height_total') is not None) or \
+          ('level_snow_height_soft' in cond[0] and  cond[0]['level_snow_height_soft'] != '' and cond[0].get('level_snow_height_soft') is not None) or \
+          ('level_place' in cond[0] and cond[0].get('level_place')  != '') or \
+          ('level_comment' in cond[0] and cond[0].get('level_comment') != '') )
```

Comment:
Isn't it enough to test if ``cond`` is not empty?
If no condition was set, has ``cond`` an initial item with blank ``level_snow_height_total`` etc.?